mappings/net/minecraft/entity/ai/goal/MoveToNearbyPointOfInterestGoal.mapping

```diff
@@ -0,0 +1 @@
+CLASS and net/minecraft/entity/ai/goal/MoveToNearbyPointOfInterestGoal
```

Comment:
Should be wandering around a point of interest.

Reply:
As far as I see, this should be a replacement for the old iron golem wander goal for fixing iron golem leaving village bug, so we should say `WanderAroundPointOfInterestGoal`